Gruntfile.js

```diff
@@ -37,15 +37,15 @@ module.exports = function( grunt ) {
 					attrs: 'fill',
 					expand: true,
 					cwd: 'sources/svg',
-					src: ['*.svg'],
+					src: [ '*.svg', '**/*.svg' ],
```

Comment:
Here you should be needing just: `src: [ '**/*.svg' ]`

As `**` already does the root folder too.